adguard/rootfs/etc/cont-init.d/adguard.sh

```diff
@@ -48,8 +49,11 @@ else
 fi
 
 # Collect IP addresses
-hosts+=($(bashio::network.ipv4_address))
-hosts+=($(bashio::network.ipv6_address))
+interfaces+=($(bashio::network.interfaces))
+for interface in "${interfaces[@]}"; do
+    hosts+=($(bashio::network.ipv4_address ${interface}))
+    hosts+=($(bashio::network.ipv6_address ${interface}))
```

Comment:
Change to prevent shell expansion

```suggestion
    hosts+=($(bashio::network.ipv4_address "${interface}"))
    hosts+=($(bashio::network.ipv6_address "${interface}"))
```